recipe/meta.yaml

```diff
@@ -1,18 +1,19 @@
 {% set name = "pyspark" %}
-{% set version = "3.1.2" %}
+{% set version = "3.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5e25ebb18756e9715f4d26848cc7e558035025da74b4fc325a0ebc05ff538e65
+  sha256: bfea06179edbfb4bc76a0f470bd3c38e12f00e1023e3ad0373558d07cff102ab
   patches:
     - 0001-Disable-symlinks-on-Windows.patch
 
 build:
-  skip: True  # [py<36]
+  # the s390x platform is not supported at this time
+  skip: True  # [py<36 or s390x]
   noarch: python
   number: 0
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
```

Comment:
There should be `python >=3.6` in `run` and `python <3.10` in `test/requires`

Reply:
> There should be `python >=3.6` in `run` and `python <3.10` in `test/requires`

the python pinnings were corrected
https://github.com/AnacondaRecipes/pyspark-feedstock/blob/update_to_3.2.0/recipe/meta.yaml#L36&&L55